package.json

```diff
@@ -44,6 +44,11 @@
         "title": "Clean Build",
         "category": "Swift"
       },
+      {
```

Comment:
You need to add the command to `commandPalette` as well. Add a "when" section to only enable for "macOS"

Reply:
TIL, thank you for pointing that out. I moved the macOS gate here based on [the docs](https://code.visualstudio.com/api/references/when-clause-contexts) and removed the `darwin` check from `command.ts` as a result.